src/Equinox/Decider.fs

```diff
@@ -53,41 +53,74 @@ type Decider<'event, 'state>
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Yield result
     member _.Transact(decide : 'state -> 'result * 'event list, ?option) : Async<'result> =
         transact option (fun (_token, state) -> async { return decide state }) (fun result _context -> result)
 
-    /// 0.  Invoke the supplied <c>_Async_</c> <c>decide</c> function with the present state, holding the <c>'result</c>
-    /// 1a. (if events yielded) Attempt to sync the yielded events events to the stream
-    /// 1b. Tries up to <c>maxAttempts</c> times in the case of a conflict, throwing <c>MaxResyncsExhaustedException</c> to signal failure.
-    /// 2.  Yield result
+    /// 1. Invoke the supplied <c>Async</c> <c>decide</c> function with the present state, holding the <c>'result</c>
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Yield result
     member _.Transact(decide : 'state -> Async<'result * 'event list>, ?option) : Async<'result> =
         transact option (fun (_token, state) -> decide state) (fun result _context -> result)
 
-    /// 0.  Invoke the supplied <c>_Async_</c> <c>decide</c> function with the present state (including extended context), holding the <c>'result</c>
-    /// 1a. (if events yielded) Attempt to sync the yielded events events to the stream
-    /// 1b. Tries up to <c>maxAttempts</c> times in the case of a conflict, throwing <c>MaxResyncsExhaustedException</c> to signal failure.
-    /// 2.  Uses <c>mapResult</c> to render the final 'view from the <c>'result</c> and/or the final <c>ISyncContext</c>
-    /// 3.  Yields the 'view
+    /// 1. Invoke the supplied <c>interpret</c> function with the present state (including extended context)
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    member _.TransactEx(interpret : ISyncContext<'state> -> 'event list, ?option) : Async<unit> =
+        transact option (fun (Context c) -> async { return (), interpret c }) (fun () _context -> ())
+
+    /// 1. Invoke the supplied <c>interpret</c> function with the present state
+    /// 2. (if events yielded) Attempt to sync the yielded events to the stream.
+    ///    (Restarts up to <c>maxAttempts</c> times with updated state per attempt, throwing <c>MaxResyncsExhaustedException</c> on failure of final attempt.)
+    /// 3. Yields a final 'view produced by <c>render</c> from the final persisted <c>ISyncContext</c>
+    member _.TransactEx(interpret : 'state -> 'event list, render : ISyncContext<'state> -> 'view, ?option) : Async<'view> =
```

Comment:
disambiguates a relatively common case where attempting to use a lambda of the form `TransactEx(fun c -> myInterpret c.State, fun c -> render c.Version, c.State)` gives a confusing error messages that ordinarily needs to be resolved by wrapping as `TransactEx((fun c -> myInterpret c.State), fun c -> render c.Version, c.State)`